experiments/list_unsupported.sh

```diff
@@ -34,8 +34,35 @@ done
 
 echo "$1: Unsupported features\n" > "$file_name".txt
 
+if ! command -v gnat > /dev/null; then
+   echo >&2 "Gnat not on PATH!"
+   exit 4
+fi
+
+ADA_HOME=`which gnat`
+ADA_HOME="$(dirname "$ADA_HOME")/.."
+DEF_ADA_INCLUDE_PATH="${ADA_HOME}/lib/gcc/x86_64-pc-linux-gnu"
+ADA_VERSION=$(find ${DEF_ADA_INCLUDE_PATH} -maxdepth 1 -type d -name '*.*.*' -print -quit)
+DEF_ADA_INCLUDE_PATH="${ADA_VERSION}/rts-native/adainclude"
+export ADA_INCLUDE_PATH="${ADA_INCLUDE_PATH:-$DEF_ADA_INCLUDE_PATH}"
+if [ ! -d "$ADA_INCLUDE_PATH" ]; then
```

Comment:
@martin-cs - There's been a bit of debate today about this particular check here, because it's perfectly valid for ADA_INCLUDE_PATH to be a list of directories, not a single directory. Two options are (a) drop this test, and just check that the variable is set, (b) somehow parse the include path variable, then check each element in the list - and also handle the case that it's also valid to have some elements in the list be non existent.... We were starting to lean towards (a).

Reply:
I removed the check for now.

Reply:
(a). check that it is set but don't try to parse.

The variable is meaningful to GNAT and to the user; we don't need to parse it.  Too much cleverness here could bite us later.  Also it is valid for the path to contain empty or non-existent directories (as long as it does have some of ones that work).